tardis/conftest.py

```diff
@@ -117,3 +119,29 @@ def included_he_atomic_data(test_data_path):
 def tardis_config_verysimple():
     return yaml_load_config_file(
         'tardis/io/tests/data/tardis_configv1_verysimple.yml')
+
+###
+# HDF Fixtures
+###
+
+@pytest.fixture(scope="session")
+def hdf_file_path(tmpdir_factory):
+    path = tmpdir_factory.mktemp('hdf_buffer').join('test.hdf')
+    return str(path)
+
+@pytest.fixture(scope="session")
+def hdf_config():
+    filename = 'tardis_configv1_verysimple.yml'
+    path = os.path.abspath(os.path.join('tardis/io/tests/data/', filename))
+    config = Configuration.from_yaml(path)
+    return config
+
+@pytest.fixture(scope="session")
+def model(hdf_config):
```

Comment:
Can you try to create a `simulation_verysimple` fixture? If that works, then we can get a model by `simulation_verysimple.model` and therefore we don't need additional fixtures.
Do you think, that could work?